src/Compilers/CSharp/Test/Symbol/SymbolDisplay/SymbolDisplayTests.cs

```diff
@@ -7584,5 +7584,26 @@ class B
                 method.ToDisplayParts(formatWithoutOptions),
                 "static void F4(nint[] x, A<nuint> y)");
         }
+
+        [Fact]
+        public void RecordDeclaration()
```

Comment:
>RecordDeclaration [](start = 20, length = 17)

In RecordTests.cs we have a bunch of IL tests with invalid Clone declarations, I think we should add a ToTestDisplayString validation to them.

Reply:
Done